ecosystem/theme-default/src/client/components/Navbar.vue

```diff
@@ -24,27 +25,20 @@ const linksWrapperStyle = computed(() => {
 })
 
 // avoid overlapping of long title and long navbar links
-onMounted(() => {
-  // TODO: migrate to css var
-  // refer to _variables.scss
-  const MOBILE_DESKTOP_BREAKPOINT = 719
+const handleLinksWrapWidth = (mobileDesktopBreakpoint: number): void => {
   const navbarHorizontalPadding =
     getCssValue(navbar.value, 'paddingLeft') +
     getCssValue(navbar.value, 'paddingRight')
-  const handleLinksWrapWidth = (): void => {
-    if (window.innerWidth < MOBILE_DESKTOP_BREAKPOINT) {
-      linksWrapperMaxWidth.value = 0
-    } else {
-      linksWrapperMaxWidth.value =
-        navbar.value!.offsetWidth -
-        navbarHorizontalPadding -
-        (navbarBrand.value?.offsetWidth || 0)
-    }
+  if (window.innerWidth < mobileDesktopBreakpoint) {
+    linksWrapperMaxWidth.value = 0
+  } else {
+    linksWrapperMaxWidth.value =
+      navbar.value!.offsetWidth -
+      navbarHorizontalPadding -
+      (navbarBrand.value?.offsetWidth || 0)
   }
-  handleLinksWrapWidth()
-  window.addEventListener('resize', handleLinksWrapWidth, false)
-  window.addEventListener('orientationchange', handleLinksWrapWidth, false)
-})
+}
+updateDeviceStatus(DeviceType.MOBILE, handleLinksWrapWidth)
```

Comment:
Maybe simplify to `updateDeviceStatus(DeviceType.MOBILE, (...) => { ... })`

Seems that we don't need to declare the callback function separately